### PR TITLE
Add SGP40 usage guide

### PR DESCRIPTION
**Pull request by rnestler**
Check the following:

 - [na] Breaking changes marked in commit message
 - [na] Changelog updated
 - [na] Code style cleaned (ran `make style-fix`)
 - [na] Tested on actual hardware

### CODE REVIEW

**Review by Zifzaf — COMMENTED** [review]
I think it's okay. Why is the hole block changed in the third commit? Did you add the hole release/sgp40 to the Makefile?

File: Makefile [hunk 1]
```diff
@@ -40,6 +40,25 @@ $(release_drivers): sgp-common/sgp_git_version.c
 	cd release && zip -r "$${pkgname}.zip" "$${pkgname}" && cd - && \
 	ln -sf $${pkgname} $@
 
+release/sgp40:
```
Comment (abrauchli) [inline Makefile:43]:
```suggestion
release/sgp40: sgp-common/sgp_git_version.c
```

Comment (abrauchli) [reply Makefile:43]: .. I think it also misses a dependecy on `docs/Application_Note_SGP40.pdf` but I don't see the make target off a quick glance

**Review by abrauchli — CHANGES_REQUESTED** [review]

File: Makefile [hunk 2]
```diff
@@ -40,6 +40,24 @@ $(release_drivers): sgp-common/sgp_git_version.c
...
+	cp -r sgp-common/* "$${pkgdir}" && \
+	cp -r $${driver}/* "$${pkgdir}" && \
+	cp docs/Application_Note_SGP40.pdf $${pkgdir} && \
+	cp CHANGELOG.md LICENSE "$${pkgdir}" && \
+	echo 'sensirion_common_dir = .' >> $${pkgdir}/user_config.inc && \
+	echo 'sgp_common_dir = .' >> $${pkgdir}/user_config.inc && \
+	echo "$${driver}_dir = ." >> $${pkgdir}/user_config.inc && \
+	cd "$${pkgdir}" && $(MAKE) $(MFLAGS) && $(MAKE) clean $(MFLAGS) && cd - && \
+	cd release && zip -r "$${pkgname}.zip" "$${pkgname}" && cd - && \
+	ln -sf $${pkgname} $@
```
Comment (abrauchli) [inline Makefile:60]: any reason you removed the blank line between the targets? I'd keep it for readability, even if they're semantically related

File: Makefile [hunk 3]
```diff
@@ -40,6 +40,24 @@ $(release_drivers): sgp-common/sgp_git_version.c
 	cd release && zip -r "$${pkgname}.zip" "$${pkgname}" && cd - && \
 	ln -sf $${pkgname} $@
 
+release/sgp40: sgp-common/sgp_git_version.c
```
Comment (abrauchli) [inline Makefile:43]: on second thought, I think it'd be wiser to replace it with the generic `prepare` target.. (also above)
```suggestion
release/sgp40: prepare
```

File: Makefile (hunk 2 above)
Comment (rnestler) [reply Makefile]: Accident. Fixed.

File: Makefile (hunk 1 above)
Comment (rnestler) [reply Makefile]: I added a docs target and added the dependencies accordingly.

File: Makefile (hunk 3 above)
Comment (rnestler) [reply Makefile]: I change it to let all release targets just depend on prepare, so fixed.

**Review by abrauchli — APPROVED** [review]

**Pull request merged**